app/containers/Applications/ApplicationCommentsContainer.tsx

```diff
@@ -43,7 +43,7 @@ export const ApplicationCommentsComponent: React.FunctionComponent<Props> = prop
               <thead style={{textAlign: 'center'}}>
```

Comment:
Each of the two comment tables should probably have some sort of distinct title describing why the comments are separated, like 'General Comments' & 'Requested Changes' or something like that.

Reply:
Added titles for columns